### PR TITLE
[Fortinet] Deprecate original integration & change logfile to filestream

### PR DESCRIPTION
**Pull request by legoguy1000**
## What does this PR do?

- Deprecate original Fortinet integration
- Swap the logfile legacy input for the filestream input

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/fortinet/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.7.3"
+  changes:
+    - description: Deprecating Fortinet package in favor of new product specific packages
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/1
```
Comment (efd6) [inline packages/fortinet/changelog.yml:6]: Please update the PR number.

File: packages/fortinet_forticlient/data_stream/log/agent/stream/filestream.yml.hbs [hunk 2]
```diff
@@ -1,24 +1,18 @@
 paths:
 {{#each paths as |path i|}}
-  - {{path}}
+ - {{path}}
```
Comment (efd6) [inline packages/fortinet_forticlient/data_stream/log/agent/stream/filestream.yml.hbs:3]: Do we need this deletion? (same for the other packages).

File: packages/fortinet_forticlient/data_stream/log/agent/stream/filestream.yml.hbs [hunk 3]
```diff
@@ -1,24 +1,18 @@
...
-fields_under_root: true
-fields:
-    observer:
-        vendor: "Fortinet"
-        product: "FortiClient"
-        type: "Anti-Virus"
 {{#contains "forwarded" tags}}
 publisher_pipeline.disable_host: true
 {{/contains}}
+prospector.scanner.exclude_files: ['\.gz$']
```
Comment (efd6) [inline packages/fortinet_forticlient/data_stream/log/agent/stream/filestream.yml.hbs:15]: Leaving this at the same line as the previous exclude reduces diff noise. (same for the other packages).

Comment (legoguy1000) [reply packages/fortinet_forticlient/data_stream/log/agent/stream/filestream.yml.hbs:15]: Ya, I copied the config from another integration so I can go back and clean it up.

**Comment by legoguy1000** [comment]
@efd6 comments resolved.

**Review by efd6 — APPROVED** [review]

**Review by andrewkroh — CHANGES_REQUESTED** [review]
I think changing from `log` to `filestream` will cause problems for users because they will lose their existing state. Much of this is discussed in https://github.com/elastic/integrations/issues/2518. I think we should hold off on this change given the packages are GA. We'll cover making the change with https://github.com/elastic/integrations/issues/2518. 

So I think the filestream changes should be reverted.

**Comment by legoguy1000** [comment]
@andrewkroh  Will do.

File: packages/fortinet/manifest.yml [hunk 4]
```diff
@@ -1,8 +1,8 @@
 name: fortinet
 title: Fortinet
-version: "1.8.0"
+version: "1.8.1"
 release: ga
-description: Collect logs from Fortinet instances with Elastic Agent.
+description: Deprecated. Collect logs from Fortinet instances with Elastic Agent.
```
Comment (andrewkroh) [inline packages/fortinet/manifest.yml:5]: Can you please check the readme of this package and see if it needs changes to reflect being deprecated. You can look at the microsoft readme to get an idea of what other deprecated pages do. I think it's a "NOTE" and title change.

File: packages/fortinet_forticlient/changelog.yml [hunk 5]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.1.0"
+  changes:
+    - description: Update Ingest Pipeline with observer Fields
+      type: enhancement # can be one of: enhancement, bugfix, breaking-change
```
Comment (andrewkroh) [inline packages/fortinet_forticlient/changelog.yml:5]: Let's drop the comment.

**Review by andrewkroh — APPROVED** [review]

**Comment by legoguy1000** [comment]
GTG

**Pull request merged**